src/lib/viewers/doc/PresentationViewer.js

```diff
@@ -313,6 +313,19 @@ class PresentationViewer extends DocBaseViewer {
             };
         };
     }
+
+    /**
+     * @override
```

Comment:
It looks like this is under the `// Private` section of the class, though it's protected/public.